locales/en.json

```diff
@@ -1,16 +1,87 @@
+            }
+        },
+        "timeline": {
+            "items": {
+                "0": {
+                    "date": "2021.06.17",
+                    "dateHeadline": "From idea to health database",
+                    "text": {
+                        "0": "On that day, LaShawn Toyoda had enough of the lack of direction from officials. A few hours later, Find a Doc is born.",
+                        "1": "This database aims to better circulate the latest information on healthcare services in Japan."
+                    },
+                    "textHeadline": "Watch our daughter, I gotta make something."
+                },
+                "1": {
+                    "date": "Presently",
+                    "dateHeadline": "A team of volunteers",
+                    "text": {
+                        "0": "Find a Doc is community-driven with the support of a team of volunteers. Currently, the team is focused on keeping the immigrant community informed about clinics offering waiting lists for vaccinations to reduce waste.",
+                        "1":  "In the future, we want Find a Doc to help non-Japanese speaking residents of Japan find clinics that offer services in their native language."
+                    },
+                    "textHeadline": "You can fill the void by citizen action.",
+                    "action": "Check out &/or join the team"
+                }
+            }
+        },
+        "trust": {
+            "cards": {
+                "0": {
+                    "text": {
+                        "0": "All links lead to official resources such as clinic and government websites.",
+                        "1": "Read them thoroughly before registering your personal information with them."
+                    }
+                },
+                "1": {
+                    "text": {
+                        "0": "Anyone in the community can submit content to the database.",
+                        "1": "As a result, is not possible for us to guarantee that accuracy. If you notice any errors, please report the issue to moderators."
+                    }
+                },
+                "2": {
+                    "text": {
+                        "0": "Our moderators try their best to detect fraudulent information.",
+                        "1": "Even so, research any service that you find on here before visiting the location or submitting your personal information to a third-party website."
+                    }
+                }
+            },
+            "footer": {
+                "headline": "This is an open-source project, and we are not liable for any damages that may be incurred from the use of Find a Doc.",
+                "license": "Read more about it in our license."
+            },
+            "header": {
+                "headline": "Can this information be trusted?",
+                "subheadline": "In short, yes, but do your due diligence"
```

Comment:
`Yes, to some degree, but do your due diligence.`

Reply:
Changed, but I wonder if 'to some degree' might rob the site of credibility?